### ISSUE TITLE
Have region name in surface layer metadata

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/brainrender-napari#97]
Naming convention for region meshes is and ending in abbreviation, e.g.`...CB_mesh`. Maybe region name should be in metadata so it displays in the tooltip?

**Comment by kira1228** [comment]
@alessandrofelder 

I don't know if I can, but can I work on this problem?

**Comment by kira1228** [comment]
Hi @alessandrofelder 

I have a rough idea of taking the `name` attribute from the following structure and adding it to the surface layer.

> STRUCTURE_TEMPLATE = {
>    "acronym": "VIS",  # shortened name of the region
>    "id": 3,  # region id
>    "name": "visual cortex",  # full region name
>    "structure_id_path": [1, 2, 3],  # path to the structure in the structures hierarchy, up to current id
>    "rgb_triplet": [255, 255, 255],  # default color for visualizing the region, feel free to leave white or randomize it
> }

Do you think this gonna work out?  🤔

**Comment by alessandrofelder** [comment]
Yes, you are welcome to tackle this, but I am not sure this issue is even possible 😢 so tackle this at your own risk ⚠ it may require major surgery to the code!

Let me clarify (and modify) the issue a bit:

This is the current situation:

![Image](https://github.com/user-attachments/assets/6ae19c63-fbe9-41f0-a8bf-b50ebb9da947)

You can see in the layer list on the lower left, that the layer name and the tooltip of the layer are the same. I don't think it is possible to change this. Maybe you can find out?

Alternatively, we might be able to change the tooltip when we hover over a surface layer in the 3D view, [similarly to what we do for 2D view currently.](https://github.com/brainglobe/brainrender-napari/blob/4e0f8356f1e7fe68a933bde8570c0e4c5db32cc0/brainrender_napari/napari_atlas_representation.py#L124) Then we could incorporate some [custom tooltip text in the layer metadata,](https://napari.org/0.5.3/guides/layers.html#layer-metadata) and display it in the 3D view. This _might_ work, but the only way to know for sure is to try :)

**Comment by alessandrofelder** [comment]
In terms of what the tooltip text should be, we suggest (for now):
* change the `Surface` layer name to be the name of the region
* make the tooltip text a formatted string like `f"{name} ({acronym}) ({atlas_name})"` so for example "Diencephalon (Di) (Eurasian Blackcap)" in the screenshot above

**Comment by kira1228** [comment]
I was thinking almost the same thing!
I'll see if I can do it as I have an idea of how to implement it. Thanks for the advice!
